demo/player.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Pillarbox - Development Page</title>
+  <link rel="icon" type="image/x-icon" href="img/favicon.ico">
+  <link rel="stylesheet" href="npm:video.js/dist/video-js.css" />
+  <link rel="stylesheet" href="scss/player.scss" />
+
+  <style>
+    body {
+      height: 100dvh;
+      margin: 0;
+      overflow: hidden;
+    }
+  </style>
+</head>
+
+<body>
+  <video id="player" class="video-js pbw-demo" controls muted></video>
+
+  <script type="module">
+    import Pillarbox from "../src/pillarbox.js";
+    import "../src/middleware/srgssr.js";
+
+    // Handle URL parameters
+    const url = new URL(location.toString());
+    const audioOnlyMode = url.searchParams.has("audioOnlyMode");
+    let currentTime = url.searchParams.get("currentTime");
+    const debug = url.searchParams.has("debug");
+    const ilHost = url.searchParams.get("ilHost") || undefined;
+    const language = url.searchParams.get("language");
+    const resize = url.searchParams.has("resize");
+    const urn = url.searchParams.get("urn") || "urn:swi:video:48115940";
```

Comment:
nit: It would also be possible to parse only the search params

```suggestion
    const searchParams = new URLSearchParams(location.search);
    const audioOnlyMode = searchParams.has("audioOnlyMode");
    let currentTime = searchParams.get("currentTime");
    const debug = searchParams.has("debug");
    const ilHost = searchParams.get("ilHost") || undefined;
    const language = searchParams.get("language");
    const resize = searchParams.has("resize");
    const urn = searchParams.get("urn") || "urn:swi:video:48115940";
```